mina-p2p-messages/src/v2/manual.rs

```diff
@@ -257,7 +257,10 @@ impl EpochSeed {
 
 impl CoinbaseStackData {
     pub fn zero() -> Self {
```

Comment:
@tizoc That method should probably be renamed to `empty` ? To make it clear that it's value is not zero

Reply:
good point

Reply:
(and same for the callers which are currently named `zero` too)

Reply:
https://github.com/openmina/openmina/pull/391